base_tier_validation/__init__.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
in all `__init__` files you should remove full header

Reply:
License must be kept: https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py